ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -3922,6 +3922,16 @@ keys:
                       type: str
                       valid_values:
                       - discard
+      event:
+        storm_control_discards:
+          enable: bool
```

Comment:
enable should include a `type` member
```
enable:
    type: bool
```

Reply:
Sorry for that inconvenience.
I will correct it tomorrow, since our servers with the development platforms are currently down.

Reply:
@hamptonmoore would this match the correct syntax?

```
  event:
    type: dict
    keys:
      storm_control_discards:
	type: dict
        keys:
	  enable:
	    type: bool
	  interval:
	    type: int
	    convert_types:
	      - str
	    min: 10
	    max: 65535
	    description: Logging interval in seconds
```

Reply:
That looks good to me! One thing I'm not sure about is if EOS could add future options to `storm-control` in the future and if it should be discards in a dict inside of there. Any thoughts @carlbuchmann?

Reply:
@hamptonmoore @carlbuchmann 
A comment from my side on this:
Currently EOS storm-control has only the action to "drop/discard" packets exceeding the configured thresholds.

Other vendors also support the action "interface shutdown" when thresholds are exceeded.
So there might be the slightly chance, that "shutdown" could be added in the future and that there would be a syslog message added for this action.

Reply:
yes, I agree with @hamptonmoore, so it would be something like this:

```yaml
  event:
    type: dict
    keys:
      storm_control:
	type: dict
        keys:
          discards:
	    type: dict
            keys:
	      enable:
	        type: bool
	      interval:
	        type: int
	        convert_types:
	          - str
	        min: 10
	        max: 65535
	        description: Logging interval in seconds
```

Reply:
@kmueller68 if you need help with schema authoring, we are also available for a live zoom call.

Reply:
@carlbuchmann thank you very much.
I will prepare things tomorrow and a short "how to do schema" in a zoom session would be very helpful.
Would you be available tomorrow morning EDT?

Reply:
Yes, I will send you an invite!